cpp/dolfin/fem/Form.h

```diff
@@ -187,6 +210,15 @@ class Form
   /// Access form integrals (const)
   const FormIntegrals& integrals() const;
 
+  // Access constants (non-const)
```

Comment:
Explain what the vector index gives, i.e. explain why this is a `std::vector` rather than a `std::map`.